advanced_survey/static/advanced-survey/src/direflow-components/field-selector/index.tsx

```diff
@@ -34,16 +34,16 @@ const FieldSelector: FC<IProps> = ({index, question}) => {
             <div className="form-group mb-2">
                 <label>Field Type:</label>
                 <select value={fieldtype} className="form-control" onChange={e => {
-                    setFieldtype(e.target.value);
+                    setFieldtype(parseInt(e.target.value));
                     setOptions({});
                 }}>
                     {Object.keys(FieldType).map((key: string) => {
                         if (isNaN(parseInt(key))) return null;
-                        return <option key={key}>{FieldType[parseInt(key)]}</option>
+                        return <option key={key} value={key}>{FieldTypeNames[parseInt(key) as FieldType]}</option>
                     })}
                 </select>
             </div>
-            {fieldtype === "TextArea" &&
+            {fieldtype === FieldType.TextArea &&
```

Comment:
Please update backend views, specially surveyjs endpoint.

Reply:
I think it was easier to just update the enum values. Does that make sense?